lib/Cpdf.php

```diff
@@ -2975,6 +3062,135 @@ protected function o_embedded_file($id, $action, $options = null): ?string
         return null;
     }
 
+    /**
+     * Enable PDF/A compliance mode
+     */
+    public function enablePdfACompliance()
+    {
+        $this->pdfa = true;
+
+        $this->o_catalog($this->catalogId, 'outputIntents', [
+            'iccProfileData' => file_get_contents(__DIR__ . '/res/sRGB2014.icc'),
```

Comment:
can this path be a variable on a line above ?
In Debian packaging I will have to change it to point to a path located in another package

Reply:
Just to make sure I understand, you'd like it to be like this?

```php
$iccProfilePath = __DIR__ . '/res/sRGB2014.icc';

$this->o_catalog($this->catalogId, 'outputIntents', [
            'iccProfileData' => file_get_contents($iccProfilePath ),
```

Or are you asking for the path to be exposed in configuration?

Reply:
That would be even easier to patch or link

For tcpdf this is what is done: https://sources.debian.org/src/tcpdf/6.7.4%2Bdfsg-1/debian/php-tcpdf.links/#L1